blaze/compute/tests/test_sql_compute.py

```diff
@@ -19,11 +19,10 @@
 
 from odo import odo, drop
 from blaze.compatibility import xfail
-from blaze.compute.sql import compute, select, lower_column, compute_up
-from blaze.expr import (
-    symbol, transform, summary, by, sin, join,
-    floor, cos, merge, nunique, mean, sum, count, exp, datetime as bz_datetime
-)
+from blaze.compute.sql import (compute, select, lower_column, compute_up,
```

Comment:
I have a style request: can we put parenthesized imports on their own lines sorted alphabetically like:

```python
from a import (
    b,
    c,
)
```

I have been updating these as I see them. The reason I prefer this style is that it makes it very easy to trace imports visually and understand where names are coming from. As it is now it is hard to quickly see what names are available, or the fact that sum is being aliased here.

If you have strong feeling about this please let me know,